viewing_party/party.py

```diff
@@ -1,23 +1,121 @@
 # ------------- WAVE 1 --------------------
 
+from tokenize import generate_tokens
+
+
 def create_movie(title, genre, rating):
-    pass
 
-# -----------------------------------------
+    if title == None or genre == None or rating == None:

```

Comment:
Good use of a guard clause here. 

When we do `title == None` we're actually checking if title is None but what we want to check is if title is empty (which is Falsy). We can accomplish that by doing
```python
    if not title or not genre or not rating: